instrumentation/opentelemetry-instrumentation-remoulade/src/opentelemetry/instrumentation/remoulade/__init__.py

```diff
@@ -0,0 +1,199 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Usage
+-----
+
+* Start broker backend
+
+::
+
+    docker run -p 5672:5672 rabbitmq
+
+* Run instrumented actor
+
+.. code-block:: python
+
+    from remoulade.brokers.rabbitmq import RabbitmqBroker
+    import remoulade
+
+    RemouladeInstrumentor().instrument()
+
+    broker = RabbitmqBroker()
+    remoulade.set_broker(broker)
+
+    @remoulade.actor
+    def multiply(x, y):
+        return x * y
+
+    broker.declare_actor(count_words)
+
+    multiply.send(43, 51)
+
+"""
+from typing import Collection, Iterable, List, Optional
+
+from remoulade import Middleware, broker
+
+from opentelemetry import trace
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.remoulade import utils
+from opentelemetry.instrumentation.remoulade.package import _instruments
+from opentelemetry.instrumentation.remoulade.version import __version__
+from opentelemetry.propagate import extract, inject
+from opentelemetry.propagators.textmap import CarrierT, Getter
+from opentelemetry.semconv.trace import SpanAttributes
+
+_REMOULADE_MESSAGE_TAG_KEY = "remoulade.action"
+_REMOULADE_MESSAGE_SEND = "send"
+_REMOULADE_MESSAGE_RUN = "run"
+
+_REMOULADE_MESSAGE_NAME_KEY = "remoulade.actor_name"
+
+
+class RemouladeGetter(Getter):
+    def get(self, carrier: CarrierT, key: str) -> Optional[str]:
+        value = carrier.get(key, None)
+        if value is None:
+            return None
+        if isinstance(value, str) or not isinstance(value, Iterable):
+            value = (value,)
+        return value
+
+    def keys(self, carrier: CarrierT) -> List[str]:
+        return []
+
+
+remoulade_getter = RemouladeGetter()
```

Comment:
I don't see why there should be new getter which does exactly what `default_getter` does